src/pool/disjoint/pool_disjoint_impl.cpp

```diff
@@ -290,8 +295,12 @@ class DisjointPool::AllocImpl {
         // Generate buckets sized such as: 64, 96, 128, 192, ..., CutOff.
         // Powers of 2 and the value halfway between the powers of 2.
         auto Size1 = params.MinBucketSize;
+        // MinBucketSize cannot be larger than CutOff.
+        Size1 = std::min(Size1, CutOff);
         // Buckets sized smaller than the bucket default size- 8 aren't needed.
         Size1 = std::max(Size1, UMF_DISJOINT_POOL_MIN_BUCKET_DEFAULT_SIZE);
+        // Calculate the exponent for MinBucketSize used for finding buckets.
+        MinBucketSizeExp = (size_t)log2(Size1);
```

Comment:
Can we use `getLeftmostSetBitPos()` here instead of `log2`.

Reply:
Done.